int8 RAISE=0
int8 LOWER=1
int8 STATUS=2
int8 shadeNumber
int8 action
---
int8 RAISED=0
int8 LOWERED=1 
int8 shadeNumber
int8 state